### PR TITLE
updating the data description in proposal

### PR DESCRIPTION
**Pull request by kmoravej**
Please review the data description.

### CODE REVIEW

File: proposal.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ Target audience: Customers Coming for Wine Tasting
 
 There is a diverse array of available wines that exist in North America. The variety alone is enough to overwhelm the common consumer of wine products; there needs to be a way to convey key information about these wines to help customers make informed selections during their wine tasting events. To do this, we will build a dashboard to help customers visualize the intricacies that disintguish different wines in North America particularily in terms of quality, price, origin and flavour. Our app will show the distribution of these factors by filtering and re-ordering on different characteristics to make direct comparisions between different types of wine. 
 
+### Section 1:Description of the data
```
Comment (nphaterp) [inline proposal.md:9]: Section 2

File: proposal.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ Target audience: Customers Coming for Wine Tasting
...
 
+In this project, we are using the [Wine Reviews](https://www.kaggle.com/zynicide/wine-reviews) Kaggle data set that contains 130k wine reviews with variety, location, winery, price, and description. The data scraped from WineEnthusiast on November 22nd, 2017, and it only contains wines with a review score higher than 80 between 1999 and 2017. For this project, we use the 54,762 instances of wine reviews from North America. 
+
+Each review consists of 13 explanatory variables that describe the wine in terms of origin (country, province, designation, region, winery), bottle information and reviews (description, points/ rating, price, title, variety), and information about the reviewers (taster name, taster Twitter handle). We are likely to select `country`, `description`, `points`,  `price`, `province`, and `variety` variables for our dashboard.
```
Comment (nphaterp) [inline proposal.md:14]: We could add another statement about what our plan is in our Data Viz. 'We plan on creating visualizations that can filter on these variables to allow consumers to explore their interests and make a selection for their wine tasting experience'

**Review by nphaterp — CHANGES_REQUESTED** [review]
I left a couple comments that we could implement if you agree

File: proposal.md (hunk 2 above)
Comment (kmoravej) [reply proposal.md:14]: > We could add another statement about what our plan is in our Data Viz. 'We plan on creating visualizations that can filter on these variables to allow consumers to explore their interests and make a selection for their wine tasting experience'

Thanks @nphaterp P for the feedback. I edit the file and update the pull request

**Review by nphaterp — APPROVED** [review]

**Pull request merged**